### PR TITLE
ethvm enhancements

### PR DESCRIPTION
**Pull request by chfast**
@winsvega's changes from #278 included.

connects to https://github.com/ethereum/libethereum/pull/278

### CODE REVIEW

File: ethvm/main.cpp [hunk 1]
```diff
@@ -258,6 +297,16 @@ int main(int argc, char** argv)
 		cout << st.json(styledJson);
 	else if (mode == Mode::OutputOnly)
 		cout << toHex(output);
+	else if (mode == Mode::Test)
+	{
+		// Output information needed for test verification and benchmarking
+		// in YAML-like dictionaly format.
+		auto exception = res.excepted != TransactionException::None;
+		cout << "output: '" << toHex(output) << "'\n";
```
Comment (winsvega) [inline ethvm/main.cpp]: '\n'

File: ethvm/main.cpp [hunk 2]
```diff
@@ -258,6 +297,16 @@ int main(int argc, char** argv)
 		cout << st.json(styledJson);
 	else if (mode == Mode::OutputOnly)
 		cout << toHex(output);
```
Comment (winsvega) [inline ethvm/main.cpp]: endl

**Comment by chfast** [comment]
Fixed. Anything else?

**Comment by winsvega** [comment]
looks good.

**Pull request merged**